Web/starter_project_barbies/pages/blog/BlogDetail.tsx

```diff
@@ -0,0 +1,137 @@
+import BlogCard from '@/components/blog/BlogCard'
+import fonts from '@/styles/blog/blog-detail.module.css'
+import { Blog } from '@/types/blog'
+import Image from 'next/image'
+
+interface BlogDetailProps {
+  blog: Blog
+}
+
+interface BlogAuthorDetailProps {
+  author: {
+    name: string
+    email: string
+    image: string
+    profession: string
+    userName: string
+  }
+}
+
+interface BlogContentProps {
+  content: string
+}
+
+interface RelatedBlogsProps {
```

Comment:
[nit] for this it is fine to reuse the BlogDetailProps as a related blog is really also a blog too

Reply:
@Adona12 has asked for the whole component to be moved to its own file. I think it's better to take the interface too instead of importing BlogDetailProps from here, even though you are right about the repetitiveness.